### PR TITLE
ManagamentTaskUpdateTest

### PR DESCRIPTION
**Pull request by pedalopon**

### CODE REVIEW

File: src/test/java/management/task/ManagementTaskUpdateTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        super.clickAndGo(By.linkText("Administrator"));
+        super.clickAndGo(By.linkText("Populate DB (samples)"));
+        super.checkAlertExists(true);
+
+    }
+    // Test cases -------------------------------------------------------------
+    @ParameterizedTest
+    @CsvFileSource(resources = "/management/task/update.csv", encoding = "utf-8", numLinesToSkip = 1)
+    @Order(10)
+    public void listAndShow(final int recordIndex, final String title,final String workload,final String description,final String isPublic, final String start, final String finish,
```
Comment (nicolasdeory) [inline src/test/java/management/task/ManagementTaskUpdateTest.java:33]: Perhaps change the method name to something more descriptivee

**Review by nicolasdeory — CHANGES_REQUESTED** [review]
image

These tests seem to be failing. Also, negative scenarios should be tested.

**Comment by nicolasdeory** [comment]
Tests still seem to be failing @pedalopon 

image
image

**Comment by pedalopon** [comment]
Positive test working fine for me. Also the negative test failing is right just is missing workload validation in workplan update.

**Comment by nicolasdeory** [comment]
Tests are working now, merging

**Pull request merged**